docs/content/integrations/embedded-elt/dlt.mdx

```diff
@@ -30,57 +40,88 @@ def example(api_key: str = dlt.secrets.value):
+
+To follow the steps in this guide, you'll need:
+
+- **To read the [dlt introduction](https://dlthub.com/docs/intro)**, if you've never worked with dlt before.
+- **To install the following libraries**:
 
-Next, we will discuss how to setup your Dagster project for _dlt_, and walk through how you can leverage a pre-defined _verified source_.
+  ```bash
+  pip install dagster dagster-embedded-elt
+  ```
 
-## Tutorial
+  Refer to the [Dagster installation](/getting-started/install) guide for more info.
 
-## 1. Configuring your Dagster project to support `dlt`
+---
+
+## Step 1: Configure your Dagster project to support dlt
+
+The first step is to define a location for the `dlt` code used for ingesting data. We recommend creating a `dlt_sources` directory at the root of your Dagster project, but this code can reside anywhere within your Python project.
 
-You will need to define a location for the `dlt` code used for ingesting data. We recommend creating a `dlt_sources` directory at the root of your Dagster project, however, this code can reside anywhere within your Python project.
+Run the following to create the `dlt_sources` directory:
 
 ```bash
 cd $DAGSTER_HOME && mkdir dlt_sources
 ```
 
-## 2. Initialize `dlt` ingestion code
+---
 
-In your `dlt_sources/` directory, you can write your own ingestion code following the [dlt tutorial](https://dlthub.com/docs/tutorial/load-data-from-an-api), or you can use a verified source. Here, we are going to demonstrate how you integrate the [GitHub source](https://dlthub.com/docs/dlt-ecosystem/verified-sources/github) provided by `dlt`.
+## Step 2: Initialize dlt ingestion code
 
-First, install the _dlt_ command line utility.
+In the `dlt_sources` directory, you can write your own ingestion code following the [dlt tutorial](https://dlthub.com/docs/tutorial/load-data-from-an-api) or you can use a verified source.
 
-```bash
-$ pip install dlt
-```
+In this example, we'll use the [GitHub source](https://dlthub.com/docs/dlt-ecosystem/verified-sources/github) provided by `dlt`.
 
-Then, create a location for your _dlt_ source code, and initialize the GitHub source.
+1. First, install the dlt command line utility by running:
 
-```bash
-$ cd dlt_sources/
-
-$ dlt init github snowflake
-Looking up the init scripts in https://github.com/dlt-hub/verified-sources.git...
-Cloning and configuring a verified source github (Source that load github issues, pull requests and reactions for a specific repository via customizable graphql query. Loads events incrementally.)
-Do you want to proceed? [Y/n]: y
-
-Verified source github was added to your project!
-* See the usage examples and code snippets to copy from github_pipeline.py
-* Add credentials for snowflake and other secrets in ./.dlt/secrets.toml
-* requirements.txt was created. Install it with:
-pip3 install -r requirements.txt
-* Read https://dlthub.com/docs/walkthroughs/create-a-pipeline for more information
-```
+   ```bash
+   pip install dlt
+   ```
```

Comment:
I added a prereqs section to this, which is standard for our guides. I feel like it would be better to add it in with installing the other libraries - what do you think?

Reply:
Actually, once a user installs `dagster-embedded-elt` `dlt` will be installed automatically. I will remove this item.

Reply:
Oh good call! I keep forgetting we do this with our integration libraries.